src/main/frontend/src/components/landingPage/ItemListLoader.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from 'react';
+import ItemList from "./ItemList";
+
+function ItemListLoader(props){
+
+    const [isLoading, setIsLoading] = useState(true);
+    const [loadedItems, setLoadedItems] = useState([]);
+
+    let content = "";
+
+    if (props.load === "newarrivals") {
+        content = "/items/getnewarrivals";
+    } else {    
+        content = "/items/getlastchance";
+    }
+
+    useEffect(() => {
+        setIsLoading(true);
+        fetch(`http://localhost:8080/api/v1${content}`
```

Comment:
Lets not fetch items like this in code, lets use itemService since it is defiend.